module_warn.lua

```diff
@@ -82,6 +142,18 @@ function Module:GetConfigTable()
             Type = bot.ConfigType.Boolean,
             Default = true
         },
+        {
+            Name = "MinimalWarnRole",
```

Comment:
Why "minimal"?

Reply:
Higher roles have the permission to warn, same for unwarn and MinimalUnwarnRole